release.yaml

```diff
@@ -186,28 +186,28 @@ docker-images:
         tag: v0.15.3
       gardenlet:
         name: eu.gcr.io/gardener-project/gardener/gardenlet
-        tag: v1.53.4
+        tag: v1.54.1
       networking-calico:
         name: eu.gcr.io/gardener-project/gardener/extensions/networking-calico
-        tag: v1.24.3
+        tag: v1.27.2
       networking-cilium:
         name: r.metal-stack.io/gardener/gardener-extension-networking-cilium
         tag: f3085c79
       scheduler:
         name: eu.gcr.io/gardener-project/gardener/scheduler
-        tag: v1.53.4
+        tag: v1.54.1
       shoot-cert-service:
         name: eu.gcr.io/gardener-project/gardener/extensions/shoot-cert-service
-        tag: v1.23.0
+        tag: v1.24.0
       shoot-dns-service:
         name: eu.gcr.io/gardener-project/gardener/extensions/shoot-dns-service
-        tag: v1.21.0
+        tag: v1.25.0
       virtual-apiserver:
         name: registry.k8s.io/kube-apiserver
-        tag: v1.23.17
+        tag: v1.24.15
       virtual-controller-manager:
         name: registry.k8s.io/kube-controller-manager
-        tag: v1.23.17
+        tag: v1.24.15
```

Comment:
For this version, we can already migrate to the community chart.

Reply:
nice, but this must then be changed in our roles ? Dunno how